src/test-utils/dom/cards/index.ts

```diff
@@ -30,6 +31,19 @@ export class CardWrapper extends ComponentWrapper {
     return this.findAllByClassName(styles.section).map(c => new CardSectionWrapper(c.getElement()));
   }
 
+  /**
+   * Returns the wrapper of the first card section that matches the specified test ID.
+   * Looks for the `data-testid` attribute that is assigned via `sections.testId` prop.
```

Comment:
`data-testid` is now part of our public API. This is for keeping the compatibility with RTL and other tools that look for `data-testid`.

This is actually one of the reasons why we decided to use this attribute rather than internal ones like `data-awsui-id` or others.

You can check the conversation in [this PR](https://github.com/cloudscape-design/components/pull/2960/files) or comments in related API proposal.